docs/swagger/spec.yml

```diff
@@ -577,6 +577,35 @@ paths:
       responses:
         204:
           description: No content, delete successful
+  /projects/{uuid}/mosaic/:
+    get:
+      summary: List of a project's scene IDs and their associated color correction parameters to be used in mosaicing
+      tags:
+        - Imagery
+      responses:
+        200:
+          description: An ordered list of scene IDs and corresponding color correction parameters
```

Comment:
These swagger definitions are missing schema's for the responses and requests (basically, just the post ones). Otherwise this API looks really good I think.